tests/unit/data_formats/test_segment_anything_format.py

```diff
@@ -0,0 +1,168 @@
+                    Bbox(
+                        0.0,
+                        0.0,
+                        0.0,
+                        0.0,
+                        label=None,
+                        id=1,
+                        attributes={
+                            "predicted_iou": 0.8607975925446022,
+                            "stability_score": 0.36768932422921985,
+                            "point_coords": [[3, 8]],
+                            "crop_box": [0.0, 0.0, 1.0, 1.0],
+                        },
+                        group=1,
+                    ),
+                    RleMask(
+                        rle={"size": [5, 10], "counts": "b1"},
+                        label=None,
+                        id=1,
+                        attributes={
+                            "predicted_iou": 0.8607975925446022,
+                            "stability_score": 0.36768932422921985,
+                            "point_coords": [[3, 8]],
+                            "crop_box": [0.0, 0.0, 1.0, 1.0],
+                        },
+                        group=1,
+                    ),
+                ],
+                attributes={"id": 2},
+            ),
+        ]
+    )
+
+    return expected_dataset
+
+
+class SegmentAnythingTest(TestDataFormatBase):
+    IMPORTER = SegmentAnythingImporter
+    EXPORTER = SegmentAnythingExporter
+    USE_TEST_CAN_EXPORT_AND_IMPORT = False
+
+    @pytest.mark.parametrize(
+        "fxt_dataset_dir",
+        [DATASET_DIR],
+    )
+    def test_can_detect(self, fxt_dataset_dir: str):
+        assert DEFAULT_ENVIRONMENT.detect_dataset(fxt_dataset_dir) == ["segment_anything"]
+
+    @pytest.mark.parametrize(
+        [
+            "fxt_dataset_dir",
+            "fxt_expected_dataset",
+            "fxt_import_kwargs",
+        ],
+        [
+            (DATASET_DIR, "fxt_dataset", {}),
+        ],
+        indirect=["fxt_expected_dataset"],
+    )
+    def test_can_import(
+        self,
+        fxt_dataset_dir: str,
+        fxt_expected_dataset: Dataset,
+        fxt_import_kwargs: Dict[str, Any],
+        request: pytest.FixtureRequest,
+    ):
+        return super().test_can_import(
+            fxt_dataset_dir,
+            fxt_expected_dataset,
+            fxt_import_kwargs,
+            request,
+        )
+
+    @pytest.mark.parametrize(
+        [
+            "fxt_expected_dataset",
+        ],
+        [
+            ("fxt_dataset",),
+        ],
+        indirect=["fxt_expected_dataset"],
+    )
+    def test_can_export_and_import(
+        self,
+        fxt_expected_dataset: Dataset,
+        test_dir: str,
+        fxt_import_kwargs: Dict[str, Any],
+        fxt_export_kwargs: Dict[str, Any],
+        request: pytest.FixtureRequest,
+    ):
+        return super().test_can_export_and_import(
+            fxt_expected_dataset,
+            test_dir,
+            fxt_import_kwargs,
+            fxt_export_kwargs,
+            request,
+        )
```

Comment:
```suggestion
    @pytest.fixture
    def fxt_dataset_dir(self) -> str:
        return get_test_asset_path("segment_anything_dataset")
    @pytest.fixture
    def fxt_expected_dataset(self) -> Dataset:
        ...
```
Since you have only one test case (fixture), it seems that we can just define fixtures and let `TestDataFormatBase` handle them.

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/1005/commits/235e5a9bb05595dfbe18cdb00211a700323dd63b